iceoryx_posh/test/moduletests/test_roudi_mempool_introspection.cpp

```diff
@@ -203,11 +203,13 @@ TEST_F(MemPoolIntrospection_test, DISABLED_send_withSubscribers)
     ChunkMock<Topic> chunk;
     mock->hasSubscribersReturn = true;
     mock->reserveSampleReturn = chunk.chunkHeader();
+    const auto& sample = chunk.sample();
 
     m_introspection.send(); /// @todo expect call to MemPoolHandler::getMemPoolInfo
 
     EXPECT_THAT(mock->deliverChunk, Eq(1));
-    EXPECT_THAT(compareMemPoolInfo(memPoolInfoContainer, chunk.sample()->m_mempoolInfo), Eq(true));
+    ASSERT_EQ(sample->size(), 1u);
+    EXPECT_THAT(compareMemPoolInfo(memPoolInfoContainer, chunk.sample()->front().m_mempoolInfo), Eq(true));
 }
```

Comment:
Technically now a test were the container contains more than one element is missing I would say (to test the new functionality), but we can leave this as is.